src/description/eth_call.md

```diff
@@ -0,0 +1,15 @@
+* **MUST NOT** affect on-chain state 
+
+* **MUST** accept a `from` account that does not exist on-chain and if `from` is not defined it **MUST** be set to `0x0000000000000000000000000000000000000000`
+ 
+* If the `to` is null or not defined on-chain it **MUST** return an empty hex string
+
+* If `max_fee_per_gas` or `max_priority_fee_per_gas` is set the other must be set manually, otherwise they both **MUST** be set to the `gasPrice` or 0 when `gasPrice` is null
```

Comment:
I think the first part of this can be captured by the OpenRPC specification, and thus we shouldn't repeat it here.  If it isn't yet captured by the OpenRPC specification then we should fix that (I thought I already did, but maybe not).

Reply:
It is incorrect to say that these should be "set to the `gasPrice`".  If the user is trying to simulate a legacy transaction, then these should **not** be set and only `gasPrice` should be set.